com.unity.ml-agents/Runtime/Policies/BrainParameters.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using UnityEngine;
 using UnityEngine.Serialization;
+using Unity.MLAgents.Actuators;
```

Comment:
needed?

Reply:
Oops, I originally had some BrainParameters->ActionSpec code here, but didn't need it.